src/composables/feature-flags.js

```diff
@@ -38,10 +38,12 @@ export default function useFeatureFlags() {
     }
   }
 
-  function reset() {
-    cheatKeys.w = false;
-    cheatKeys.f = false;
-    features.value['new-web-forms'] = false;
+  function reset(event) {
+    if (event.target.classList.contains('btn-web-form')) {
```

Comment:
What happens if you click on the icon in the button? I think the icon would be the target, so this will evaluate to `false` even though we do want to reset. How about:

```js
if (event.target.closest('.btn-web-form') != null) {
```

Reply:
When I click on the icon, the button gets the focus because `span` by default is not focusable.

Reply:
Ah good point.